.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+# See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  Package:
+    runs-on: ubuntu-latest
+    container: registry.opensuse.org/yast/head/containers/yast-cpp:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v2
+
+    - name: Prepare System
+      run: |
+        zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
```

Remove the `--gpg-auto-import-keys` option
```suggestion
        zypper --non-interactive in --no-recommends \
```